src/modules/dashboard/ui/components/dashboard-sidebar.tsx

```
@@ -0,0 +1,113 @@
+          <p className="text-2xl font-semibold">Meet.AI</p>
+        </Link>
+      </SidebarHeader>
+      <div className="px-4 py-2">
+        <Separator className="opacity-10 text-[#5D6B68]/10" />
+      </div>
+      <SidebarContent>
+        <SidebarGroup>
+          <SidebarGroupContent>
+            <SidebarMenu>
+              {firstSection.map((item) => (
+                <SidebarMenuItem>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)